Gemfile.tt

```diff
@@ -4,35 +4,54 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby File.read(".ruby-version")
 
 gem "rails", "<%= Rails.version %>"
-gem "puma", "~> 4.1"
-gem "pg"<%= gemfile_requirement("pg") || ', "~> 0.18"' %>
+gem "puma"
+gem "pg"
 gem 'dotenv-rails', require: "dotenv/rails-now"
-gem "bootsnap"<%= gemfile_requirement("bootsnap") %>, require: false
-gem "sassc-rails"
```

Comment:
`sassc-rails` no longer required because we build all our SCSS in webpack in new apps. Also `sassc-rails` is often annoying to install because of it's C dependency.